[WIP] Remove esbuild from dependencies (#5483)
microsoft/BotFramework-WebChat: CHANGELOG.md

- Added support for liner message activities in the Fluent theme, in PR [#5272](https://github.com/microsoft/BotFramework-WebChat/pull/5272), by [@OEvgeny](https://github.com/OEvgeny)
- Added new `uiState` props and `useUIState` hook, in PR [#5276](https://github.com/microsoft/BotFramework-WebChat/pull/5276), by [@compulim](https://github.com/compulim)
- (Experimental) Added support for using Web Chat inside a shadow DOM, in PR [#5196](https://github.com/microsoft/BotFramework-WebChat/pull/5196), by [@OEvgeny](https://github.com/OEvgeny)

> 
> ### Removed
> 
> - Fixes [#5433](https://github.com/microsoft/BotFramework-WebChat/issues/5433). Removed dependency on `abort-controller-es5` in favor of the standard `abort-controller` package to prevent multiple versions of esbuild from being installed, in PR [#XXX](https://github.com/microsoft/BotFramework-WebChat/pull/XXX)

Reviewer: ```suggestion
```

@copilot move this to the existing "Removed" section at line 231

- Added support for liner message activities in the Fluent theme, in PR [#5272](https://github.com/microsoft/BotFramework-WebChat/pull/5272), by [@OEvgeny](https://github.com/OEvgeny)
- Added new `uiState` props and `useUIState` hook, in PR [#5276](https://github.com/microsoft/BotFramework-WebChat/pull/5276), by [@compulim](https://github.com/compulim)
- (Experimental) Added support for using Web Chat inside a shadow DOM, in PR [#5196](https://github.com/microsoft/BotFramework-WebChat/pull/5196), by [@OEvgeny](https://github.com/OEvgeny)

> 
> 

Reviewer: ```suggestion

```

# Removed

- Deprecating `disabled` props and `useDisabled` hook in favor of new `uiState` props and `useUIState` hook, in PR [#5276](https://github.com/microsoft/BotFramework-WebChat/pull/5276), by [@compulim](https://github.com/compulim)

> - Fixes [#5433](https://github.com/microsoft/BotFramework-WebChat/issues/5433). Removed dependency on `abort-controller-es5` in favor of the standard `abort-controller` package to prevent multiple versions of esbuild from being installed, in PR [#XXX](https://github.com/microsoft/BotFramework-WebChat/pull/XXX)

Reviewer: ```suggestion
- Fixes [#5433](https://github.com/microsoft/BotFramework-WebChat/issues/5433). Removed dependency on `abort-controller-es5` in favor of the standard `abort-controller` package to prevent multiple versions of esbuild from being installed, in PR [#5483](https://github.com/microsoft/BotFramework-WebChat/pull/5483), by [@copilot](https://github.com/copilot)
```
